test/resourceNameTest.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Story = require('../model/constructor.js');
+const superagent = require('superagent');
+const storage = require('../lib/storage.js');
+const apiURL = 'http://localhost:3000';
+
+describe('testing constructor model function', function (){
+  it('should return a story with text', function (){
+    let data = {
+      name: 'The empire strikes back',
+      text: 'Once upon a time in a galazy far, far, away',
+    };
+    let starwars = new Story(data);
+    expect(Boolean(starwars.id)).to.equal(true);
+    expect(starwars.name).to.equal(data.name);
+    expect(starwars.text).to.equal(data.text);
+  });
+}); //end of describe block
+
+//****************Testing POST here******************************************
+describe('testing /api/story' , function(){
```

Comment:
-1 point

I'm getting an error because you haven't required the server in so that the server can turn on. We need that in order to get your POST and GET tests to run properly. 

You can do that by addding ``require('../server.js')`` to line 8 at the top of your test module.